edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/NestedDeviceScopeApiClient.cs

```diff
@@ -194,6 +199,7 @@ async Task<ScopeResult> GetIdentitiesInTargetScopeInternalAsync(Uri uri, Option<
             HttpClient client = this.proxy
                 .Map(p => new HttpClient(new HttpClientHandler { Proxy = p }, disposeHandler: true))
                 .GetOrElse(() => new HttpClient());
+            client.Timeout = HttpOperationTimeout;
```

Comment:
Do we already have tests surrounding timeout?

Reply:
not unit tests, since this is using HttpClient can't be mocked. This issue was found in nested connectivity tests, but is not a specific test for this case, it only happened when multiple l3 edge devices started in the same time and making calls to the parent and those calls timed out. The call to parent timing out was also caused by another issue which was fixed #4817